wafer_space/projects/tests/test_content_processors.py

```
@@ -0,0 +1,136 @@
+
+    class IncompleteProcessor(ContentProcessor):
+        pass
+
+    with pytest.raises(TypeError, match="Can't instantiate abstract class"):
+        IncompleteProcessor()  # type: ignore[abstract]
+
+
+class MockProcessor(ContentProcessor):
+    """Mock processor for testing."""
+
+    def can_process(self, filename: str, file_path: Path) -> bool:
```

Ruff (0.14.5)

Unused method argument: `file_path`

(ARG002)